src/tags.rs

```diff
@@ -9,39 +9,43 @@ use diesel::prelude::*;
 
 /// Remove a key value pair from the tags.  
 pub fn delete(args: Args) -> Result<()> {
-    let conn = DB.get()?;
-    let key = args
-        .params
-        .get("key")
-        .ok_or("Unable to retrieve param: key")?;
-
-    match diesel::delete(tags::table.filter(tags::key.eq(key))).execute(&conn) {
-        Ok(_) => args.msg.react(args.cx, "✅")?,
-        Err(_) => args.msg.react(args.cx, "❌")?,
+    if api::is_wg_and_teams(&args)? {
+        let conn = DB.get()?;
+        let key = args
+            .params
+            .get("key")
+            .ok_or("Unable to retrieve param: key")?;
+
+        match diesel::delete(tags::table.filter(tags::key.eq(key))).execute(&conn) {
+            Ok(_) => args.msg.react(args.cx, "✅")?,
+            Err(_) => args.msg.react(args.cx, "❌")?,
```

Comment:
I think we should take the opportunity to have an explicit error message here. Acknowledging that everything went as intended can be done via a reaction, but errors should be explained.